src/containers/Editor/components.tsx

```diff
@@ -196,14 +213,52 @@ type EditorTitleProps = {
 };
 
 const EditorTitle: React.FC<EditorTitleProps> = ({ type }) => {
+
+  const {
+    active,
+    project,
+    mutator,
+    deleteContract,
+  } = useProject();
+
+  const projectPath = isUUUID(project.id) ? project.id : "local";
+
   return (
-    <Heading>
-      {type === EntityType.Account && "Contract"}
-      {type === EntityType.TransactionTemplate && "Transaction Template"}
-      {type === EntityType.ScriptTemplate && "Script Template"}
+    <HeadingContainer>
+      <Heading>
+        {type === EntityType.Account && "Contracts"}
+        {type === EntityType.TransactionTemplate && "Transaction Template"}
+        {type === EntityType.ScriptTemplate && "Script Template"}
+
+        <Version/>
+      </Heading>
+      {type === EntityType.Account && <ContractList
+          title="Contracts"
 
-      <Version/>
-    </Heading>
+          //soe          
+          items={
+            project.contracts.filter(item => item.index === active.index)
+          }
+          active={
+            active.type == EntityType.Account ? active.index : null
+          }
+          onSelect={(_, id) => {
+            navigate(`/${projectPath}?type=account&id=${project.accounts[active.index].id}&contractId=${id}`)
```

Comment:
We can probably move definition of `project.accounts[active.index].id` somewhere above for better code readability.
That field is being accessed at least 4 times in this `return` block